source/drivers/Makefile

```
@@ -0,0 +1,234 @@
+
+# Path to the kernel base file used as input by MKNEXROM. If NEXTOR_BASE
+# is set, use it as-is. Otherwise build it via the kernel Makefile and
+# stash a local copy named after the active variant.
+
+ifeq ($(strip $(NEXTOR_BASE)),)
+
+LOCAL_BASE := Nextor-$(VERSION).base$(_VARIANT_SUFFIX).dat
+
+base: $(LOCAL_BASE)
+
+$(LOCAL_BASE):
```

checkmake (0.3.2)

[warning] Target body for "$(LOCAL_BASE)" exceeds allowed length of 5 lines (8).

(maxbodylength)